scripts/bootstrap

```diff
@@ -15,13 +15,15 @@ Pull down secrets from S3.
 "
 }
 
-# TODO: potentially as a part of this issue: https://github.com/azavea/echo-locator/issues/376
-# after switching to a different bundler, we may want to copy and populate some env vars
 function pull_env() {
     pushd "${DIR}"
 
-    echo "Pulling .env from ${1}"
-    # aws s3 cp "s3://${1}/.env" ".env"
+    if aws s3 cp "s3://${ECHOLOCATOR_SETTINGS_BUCKET}/.env" ".env"; then
```

Comment:
If I run this with a bare `./scripts/setup` I get the following output:
```
fatal error: Parameter validation failed:
Invalid bucket name "": Bucket name must match the regex "^[a-zA-Z0-9.\-_]{1,255}$" or be an ARN matching the regex "^arn:(aws).*:(s3|s3-object-lambda):[a-z\-0-9]*:[0-9]{12}:accesspoint[/:][a-zA-Z0-9\-.]{1,63}$|^arn:(aws).*:s3-outposts:[a-z\-0-9]+:[0-9]{12}:outpost[/:][a-zA-Z0-9\-]{1,63}[/:]accesspoint[/:][a-zA-Z0-9\-]{1,63}$"
```

I think maybe this just needs to check for the presence of the `ECHOLOCATOR_SETTINGS_BUCKET` environment variable before attempting to download? And then the `.env` check should work fine as-is so that if someone already has a `.env` file it won't do anything and if they don't it'll copy them one from the sample.

Reply:
That is a great point. I modified the code block in there so that:
- if there is already a dotenv file, do not bother do anything extra
- otherwise:
      - if no S3 bucket provided as the env var, copy from the sample dotenv
      - if has S3 bucket provided as env var, pull from S3

This is reflected here: https://github.com/azavea/echo-locator/pull/500/commits/5cd4b0c4e4a282717610e36c8b4a32f16f5a2124